html/browsers/browsing-the-web/history-traversal/persisted-user-state-restoration/scroll-restoration-fragment-scrolling-cross-origin.html

```diff
@@ -11,57 +11,63 @@
   }
 </style>
 <div id="log"></div>
-<iframe></iframe>
 <script>
   'use strict';
 
-  // The test does the following navigation steps for iframe
-  // 1. load page-with-fragment.html#fragment
-  // 2. load blank1
-  // 3. go back to page-with-fragment.html
-  async_test(function(t) {
-    var iframe = document.querySelector('iframe');
-    var hostInfo = get_host_info();
-    var basePath = location.pathname.substring(0, location.pathname.lastIndexOf('/'));
-    var localURL = hostInfo.HTTP_ORIGIN + basePath + '/resources/page-with-fragment.html#fragment';
-    var remoteURL = hostInfo.HTTP_REMOTE_ORIGIN + basePath + "/resources/blank1.html"
+  var next;
+  function frameOnload() {
+    if (next) {
+      next();
+      dump("next \n");
+    } else {
+      dump("no next \n");
```

Comment:
@jgraham @smaug---- `dump()` is not standard - at least Chrome broke: https://wpt.fyi/results/html/browsers/browsing-the-web/history-traversal/persisted-user-state-restoration/scroll-restoration-fragment-scrolling-cross-origin.html?diff&filter=ADC&run_id=290740005&run_id=269190009